Example/CardParts/CustomMarginsCardController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CustomMarginsCardController.swift
+//  CardParts_Example
+//
+//  Created by Jonathan Jordan on 12/3/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import CardParts
+
+class CustomMarginsCardController: CardPartsViewController, CustomMarginCardTrait {
```

Comment:
Could you just move the `CustomMarginCardTrait` to an extension of `CustomMarginsCardController` like [this example](https://github.com/intuit/CardParts/blob/master/Example/CardParts/ThemedCardController.swift#L52).  Besides that the PR looks great - just tested it!  We will merge once this and @badrinathvm change is complete.

Reply:
Thanks! The requested changes have been completed.